resources/public/blank.html

```diff
@@ -2,27 +2,29 @@
 <html>
     <head lang="en">
         <meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>OrcPub</title>
-	<link rel="stylesheet" href="/css/style.css" type="text/css">
-	<link rel="stylesheet" href="/css/compiled/styles.css" type="text/css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="The most advanced Dungeons & Dragons 5th Edition (D&D 5e) character builder/generator in the multiverse.">
+        <title>The New OrcPub: D&D 5e Character Builder/Generator</title>
+		<link rel="stylesheet" href="/css/style.css" type="text/css">
+		<link rel="stylesheet" href="/css/compiled/styles.css" type="text/css">
+	<script>
+	  (function(d, s, id){
+	  var js, fjs = d.getElementsByTagName(s)[0];
+	  if (d.getElementById(id)) {return;}
+	  js = d.createElement(s); js.id = id;
+	  js.src = "//connect.facebook.net/en_US/sdk.js";
```

Comment:
Are you adding Facebook here? What is this SDK being used for?

Reply:
Future use? It was there. The index.html and blank.html only fire on figwheel configs... they aren't used when the app is run as a uberjar.